apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/dot-palette-input-filter/dot-palette-input-filter.component.html

```diff
@@ -0,0 +1,12 @@
+<button *ngIf="goBackBtn" data-testId="goBackBtn" (click)="goBack.emit()">
+    <dot-icon data-testId="goBack" name="keyboard_arrow_left" size="18"></dot-icon>
+</button>
```

Comment:
It doesn't make sense that lives here, it should just next to `<dot-palette-input-filter>` where is used.